plugins/services/src/js/components/forms/MultiContainerVolumesFormSection.tsx

```diff
@@ -24,7 +24,7 @@ import { getContainerNameWithIcon } from "../../utils/ServiceConfigDisplayUtil";
 import { FormReducer as volumeMounts } from "../../reducers/serviceForm/MultiContainerVolumes";
 import VolumeConstants from "../../constants/VolumeConstants";
 
-const excludedTypes = ["DSS", "EXTERNAL"];
+const excludedTypes = ["DSS", "EXTERNAL", "EXTERNAL_CSI"];
```

Comment:
we don't want the option to appear for pods. i'll add an integration test for that later.